### PR TITLE
MSTR-286: Add support to Braintree sdk client to store credit cards

### PR DESCRIPTION
**Pull request by pcandia**

### CODE REVIEW

File: src/apps/myaccount/submodules/billing/billing.js [hunk 1]
```diff
@@ -161,12 +177,37 @@ define(function(require) {
 				if (!settingsItem.hasClass('open')) {
 					settingsItem.find('input').keyup();
 				}
+
+				template.find('.credit-card-container').show();
+			});
+
+			template.on('change', '#dropin_container', function(e) {
+				console.log('changed');
```
Comment (masmerino13) [inline src/apps/myaccount/submodules/billing/billing.js:185]: remove this, also, is this whole block of code needed?

File: src/apps/myaccount/submodules/billing/billing.js [hunk 2]
```diff
@@ -88,71 +151,24 @@ define(function(require) {
...
 
-			template.find('.edit-credit-card').on('click', function(e) {
+			template.on('click', '.braintree-toggle', function(e) {
 				e.preventDefault();
 
-				template.find('.edition').show();
-				template.find('.uneditable').hide();
-				displayCardType('');
+				template.find('.save-card').addClass('show');
+				//template.find('.uneditable').hide();
```
Comment (masmerino13) [inline src/apps/myaccount/submodules/billing/billing.js:164]: remove if not needed

File: src/apps/myaccount/submodules/billing/billing.js [hunk 3]
```diff
@@ -88,71 +151,24 @@ define(function(require) {
...
+				e.preventDefault();
 
-			template.find('#credit_card_number').on('paste', function(e) {
-				var currentElement = $(this);
-				//Hack for paste event: w/o timeout, the value is set to undefined...
-				setTimeout(function() {
-					displayCardType(currentElement.val());
-				}, 0);
+				template.find('.save-card').removeClass('show');
+				//template.find('.uneditable').show();
```
Comment (masmerino13) [inline src/apps/myaccount/submodules/billing/billing.js:171]: same

File: src/apps/myaccount/submodules/billing/billing.js [hunk 4]
```diff
@@ -40,28 +40,86 @@ define(function(require) {
...
-						name: 'layout',
-						data: results,
-						submodule: 'billing'
-					}));
+							name: 'layout',
+							data: results,
+							submodule: 'billing'
+						})),
+						setCardHeader = function(creditCard, button) {
+							var cardType = _.toLower(creditCard.card_type) === 'american express'
```
Comment (masmerino13) [inline src/apps/myaccount/submodules/billing/billing.js:63]: not really needed to use lodash#toLower, native js .toLowerCase() will provide bette rperformace and same result

**Review by masmerino13 — COMMENTED** [review]
I just added few comments/suggestions

File: src/apps/myaccount/submodules/billing/billing.js (hunk 1 above)
Comment (pcandia) [reply src/apps/myaccount/submodules/billing/billing.js:185]: good point, i forgot to remove it :(

**Review by guillegr123 — DISMISSED** [review]
Lgtm!

**Review by masmerino13 — DISMISSED** [review]

**Review by guillegr123 — APPROVED** [review]

**Review by guillegr123 — APPROVED** [review]

**Pull request merged**